src/napari_epitools/widgets/__init__.py

```diff
@@ -0,0 +1,3 @@
+from napari_epitools.widgets.cell_statistics import create_cell_statistics_widget
+from napari_epitools.widgets.projection import create_projection_widget
+from napari_epitools.widgets.segmentation import create_segmentation_widget
```

Comment:
In the section [You should know sub-packages exist](https://blog.nicholdav.info//four-tips-structuring-research-python/#you-should-know-sub-packages-exist), it [explains why](https://blog.nicholdav.info//four-tips-structuring-research-python/#you-should-know-sub-packages-exist:~:text=This%20kind%20of,your%20API%20too.) you would import specific functions into a sub-package's `__init__.py` file - to make imports more concise (e.g. `napari_epitools.widgets.create_cell_statistics_widget` vs `napari_epitools.widgets.cell_statistics.create_cell_statistics_widget`), and to hide internal functions that shouldn't be accessed from outside of the sub-package (e.g. there's no need to use `napari_epitools.widgets.cell_statistics._create_colour_labels_widgets` outside of the sub-package) . But the function names could probably be a little more concise

Reply:
I understand the second bit. I still find the first bit contradicts itself somewhere else in that blog about:
- not bloating the namespace
- also this bit https://blog.nicholdav.info//four-tips-structuring-research-python/#import-modules-not-functions-and-classes-for-readability
- and also, either something you said or in the blog that I can't find, it suggessted it would be better to do `import a.b.c` and then run the code as `a.b.c.d()` so that you know exactly where the function `d` is from. Rather than, e.g. `from a.b.c import d`. To me, this seems like the same kind of thing.

Essentially, in learning all this stuff it makes me see the many flaws in Python 😆